bouncer/shared/upgrade_network.ts

```diff
@@ -266,14 +313,18 @@ export async function upgradeNetworkGit(
   const isCompatible = isCompatibleWith(fromTomlVersion, newToTomlVersion);
   console.log('Is compatible: ' + isCompatible);
 
+  const localnetInitPath = `${currentVersionWorkspacePath}/localnet/init`;
   if (isCompatible) {
     console.log('The versions are compatible.');
     await simpleRuntimeUpgrade(nextVersionWorkspacePath, true);
+
+    // TODO: Add restart nodes support, as in the prebuilt case.
```

Comment:
leaving this as a TODO because I'm considering deleting the upgrade test via git, feels a bit unnecessary now